tests/visualization_tests/test_edf.py

```diff
@@ -28,27 +39,39 @@ def test_plot_optimization_history(direction: str) -> None:
     study0 = create_study(direction=direction)
     study0.optimize(lambda t: t.suggest_float("x", 0, 5), n_trials=10)
     figure = plot_edf(study0)
+    data = figure.data
+    _validate(data[0]["y"])
```

Comment:
The `data` variable is only used in a single line. How about simplifying it as follows?
```suggestion
    _validate(figure.data[0]["y"])
```